bridge/engine/ActiveMediaList.cpp

```diff
@@ -355,12 +384,35 @@ void ActiveMediaList::process(const uint64_t timestampMs, bool& outDominantSpeak
 
     auto dominantSpeaker = heap.top();
 
+    if (!_c9_conference)
+    {
+        _activeTalkers.clear();
+    }
+
     for (size_t i = 0; i < _audioLastN && !heap.empty(); ++i)
     {
-        updateActiveAudioList(heap.top().participant);
+        auto top = heap.top();
+        updateActiveAudioList(top.participant);
+        if (!_c9_conference)
```

Comment:
An interesting angle on this. What if we have a hybrid conference, in the name of integration. Some users produce the ptt and some do not.
Any user that is not producing c9 hdr would be excluded from the active talker list.

Reply:
We've discussed it with Shanling, and come to conclusion that mixed conference will never happen. It's in the doc I shared.